### DISCUSSION TITLE
Tool-инвентарь

### DISCUSSION THREAD

**Discussion by FSBJI** [octothorp-team/public-dobrograd#760] [Предложения]
Добавить его всем игрокам для использования. + Тулл прогресс

**Comment by Dawaren** [comment]
Не знаю насчет прогресса, но инвентарем будут абузить. Просто не будут покупать ящики, а поставят маленький проп с весом в 999 литров.

**Comment by Emberenius** [comment]
Возможно и стоит, но тогда будут обесценены ящики. Даже если и делать, то поставить лимит в 1 применение и, наверное, литров 50

**Comment by Wani4ka** [comment]
Попросил бы сразу всю гмпанель игрокам выдать, чего уж


Выше уже прекрасно описали, что произойдёт, если выдать тул "инвентарь" всем игрокам. Даже с лимитом в 1 использование, это будет халявная коробка на 50л с возможностью крафта. Никто игрокам такое давать не будет

Что касается "Прогресса". Случился однажды баг, в результате которого этот тул стал доступен игрокам. Кхм.
![unknown.png](https://user-images.githubusercontent.com/31739611/143538540-cd3dae91-8faf-4724-aeb3-fc624baff202.png)

Суммируя: Ролевикам и Ролевикам+ может быть ещё можно доверить эти тулы, но делать так, чтобы любой игрок с 0 часов мог ставить халявную коробку и делать секс-кукол, даже при условии ограничения этих тулов на плюшки "Строитель" или "Рассказчик" - нет уж, спасибо

**Comment by outclown** [comment]
> Добавить его всем игрокам для использования. + Тулл прогресс

Это будут абузить, при какой-то ситуации плачевной для персонажа, будь то угроза ограбления или смерти, человек будет прятать проп с инвентарем в стену, или сделает его невидимым и поставит в пропы, если это как-то ограничать - я за
